### ISSUE TITLE
Using a java argument with a full windows path "C:\..." breaks due to the ":" delimiter

### ISSUE THREAD

**Issue by Frontear** [PrismLauncher/PrismLauncher#1494]
### Operating System

Windows

### Version of Prism Launcher

Prism Launcher 7.2

### Version of Qt

Qt 6.5.1

### Description of bug

There's a bit of an issue with GLFW in LWJGL being outdated and sometimes causing segmentation faults with "EXCEPTION_ACCESS_VIOLATION" when trying to invoke `nglfwCreateWindow`. Since this issue is so common, especially within users of our modpacks, I've created a repository dedicated to hosting pre-compiled GLFW libraries over at https://github.com/Frontear/glfw/releases so that users can easily override them and launch minecraft.

The way to do this is to use a java argument `-Dorg.lwjgl.glfw.libname=path/to/glfw3.dll`. Prism seems to do something weird that others launchers (ive tested on ATLauncher) do not, and that results in Prism crashing with a `java.nio.file.InvalidPathException: Illegal char <:> at` due to the delimiter in the drive specifier on windows. Again, **ATLauncher does NOT have this issue**, so its very obviously a Prism specific issue. **HOWEVER**, this issue goes away **IF** I remove any quotes wrapping the path.

Reproduction steps:
- Create a fresh instance of (for this case, 1.20.1 and forge 47.1.3), BUT this issue affects VANILLA AND FABRIC too.
- Probably download or compile by self, a glfw3.dll file for testing
- In Settings -> Java arguments, add `-Dorg.lwjgl.glfw.libname="C:\Users\YOUR_USERNAME\Downloads\glfw3.dll"` or wherever else you save the dll.
- Run Minecraft, it will crash. I will upload the full log here: https://mclo.gs/EujxM39

ATLauncher and similar launchers do NOT crash with this error, so Prism is in some way mangling the path somewhere in the launch process that is breaking the parameter. Funny enough, if I **remove** the quotations wrapped around the path, ie. make the argument `-Dorg.lwjgl.glfw.libname=C:\Users\Frontear\Downloads\glfw3.dll`, minecraft will launch successfully. This is consistent with other launchers as well.

Now why is this a problem? Just don't wrap with quotes. Well the problem originates IF the user has spaces in their path, most commonly in their username itself (this is an issue that one of my users has, with their username containing a space "Adrian Lorin", and thus completely breaking. In the end I had to ask them to move the glfw3.dll to their C:\ root to prevent the spaces breaking it).

Now lets say, for testing purposes, we move the dll to a folder with a space (im going to make a new folder in Downloads with the name "spa ce" for this)

Reproduction steps:
- Create a fresh instance (again for this case, 1.20.1 with forge 47.1.3, but again, this affects ALL minecraft versions and loaders)
- Download/compile the glfw3.dll
- Settings -> Java arguments, add `-Dorg.lwjgl.glfw.libname=C:\Users\YOUR_USERNAME\Downloads\spa ce\glfw3.dll`
- Minecraft will crash pre-loading phase because the java parameter is invalid since there is a space character in the path, log here: https://mclo.gs/FsuIm1h

ATLauncher, from my testing, does NOT have this issue (though I needed to wrap the path in quotes otherwise it would have crashed). With Prism I don't have the option to wrap with quotes because it crashes 100% with quotes. 

Please fix this issue, I know its a very random and not very often encountered issue, but it breaks the consistency of what I would otherwise consider the best MC launcher. Specifically, all I'm hoping to fix is the crash with using quotations. Fixing that would simultaneously fix the issue with spaces in path, since quote wrapping would resolve that.

### Steps to reproduce

Please see above ^

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by TheKodeToad** [comment]
have you tried `"-Dorg.lwjgl.glfw.libname=C:\Users\YOUR_USERNAME\Downloads\spa ce\glfw3.dll"`?

**Comment by epicacrylic** [comment]
Had the same error, but on 7.1

The launcher doesn't like spaces in the address for the DLL but once you remove the spaces it works fine. Dunno if the options don't know how to read quotes but I tried with a folder that has a space and it either tried to read it as a separate library or said the location was incorrect because of a colon.

But by removing the spaces and not having quotes using `-Dorg.lwjgl.glfw.libname=` worked

**Comment by Ryex** [comment]
quote the entire argument `"-Dorg.lwjgl.glfw.libname=C:\Users\YOUR_USERNAME\Downloads\spa ce\glfw3.dll"` as KodeTode suggests and it should word correctly

**Comment by Frontear** [comment]
It does, can confirm, but why does just quoting the path break?

**Comment by Frontear** [comment]
Nope, just tested on a different instance, and it breaks even with wrapping the entire command in quotes.

https://mclo.gs/5zIDQEi

This was running SteamPunk on Curseforge, v6.

EDIT:
it seems prism is dropping the slashes? I'm using backward slashes as is normal on Windows. the full jvm args are:

```
-XX:+UnlockExperimentalVMOptions -XX:+UnlockDiagnosticVMOptions -XX:+AlwaysActAsServerClassMachine -XX:+AlwaysPreTouch -XX:+DisableExplicitGC -XX:+UseNUMA -XX:NmethodSweepActivity=1 -XX:ReservedCodeCacheSize=400M -XX:NonNMethodCodeHeapSize=12M -XX:ProfiledCodeHeapSize=194M -XX:NonProfiledCodeHeapSize=194M -XX:-DontCompileHugeMethods -XX:MaxNodeLimit=240000 -XX:NodeLimitFudgeFactor=8000 -XX:+UseVectorCmov -XX:+PerfDisableSharedMem -XX:+UseFastUnorderedTimeStamps -XX:+UseCriticalJavaThreadPriority -XX:ThreadPriorityPolicy=1 -XX:AllocatePrefetchStyle=3 -XX:+UseG1GC -XX:MaxGCPauseMillis=37 -XX:+PerfDisableSharedMem -XX:G1HeapRegionSize=16M -XX:G1NewSizePercent=23 -XX:G1ReservePercent=20 -XX:SurvivorRatio=32 -XX:G1MixedGCCountTarget=3 -XX:G1HeapWastePercent=20 -XX:InitiatingHeapOccupancyPercent=10 -XX:G1RSetUpdatingPauseTimePercent=0 -XX:MaxTenuringThreshold=1 -XX:G1SATBBufferEnqueueingThresholdPercent=30 -XX:G1ConcMarkStepDurationMillis=5.0 -XX:G1ConcRSHotCardLimit=16 -XX:GCTimeRatio=99 -Djava.security.manager=allow -Dfile.encoding=UTF-8 --add-opens java.base/jdk.internal.loader=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED --add-opens java.base/java.text=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/jdk.internal.reflect=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens jdk.naming.dns/com.sun.jndi.dns=ALL-UNNAMED,java.naming --add-opens java.desktop/sun.awt.image=ALL-UNNAMED --add-modules jdk.dynalink --add-opens jdk.dynalink/jdk.dynalink.beans=ALL-UNNAMED --add-modules java.sql.rowset --add-opens java.sql.rowset/javax.sql.rowset.serial=ALL-UNNAMED --add-exports java.base/sun.security.util=ALL-UNNAMED --add-opens java.base/java.util.jar=ALL-UNNAMED "-Dorg.lwjgl.glfw.libname=C:\Users\Frontear\AppData\Roaming\PrismLauncher\natives\glfw3.dll"
```

**Comment by Frontear** [comment]
> Had the same error, but on 7.1
> 
> 
> 
> The launcher doesn't like spaces in the address for the DLL but once you remove the spaces it works fine. Dunno if the options don't know how to read quotes but I tried with a folder that has a space and it either tried to read it as a separate library or said the location was incorrect because of a colon.
> 
> 
> 
> But by removing the spaces and not having quotes using `-Dorg.lwjgl.glfw.libname=` worked

Yeah this is my exact experience as well. Problem is this breaks consistency across MC launchers and the recommended solution of wrapping the entire arg in quotes: "-Dorg...=..." does not work either, as in my recent experience

**Comment by Aces-and-Jacks** [comment]
Would this still be a fixable issue for prism? Or closed because...

(I don't use windows but I feel closing issues that still persist would bury them)

**Comment by Octol1ttle** [comment]
Does this issue still occur on 11.0.2?
